### PR TITLE
ROX-10990: Split Cluster Overview Findings tabs into 3 CVE types

### PR DESCRIPTION
**Pull request by vjwilson**
## Description

This splits the CVEs in the Cluster Overview Findings section into three types, each in its own tab, when the feature flag is on.

## Checklist
- [ ] Investigated and inspected CI test results

## Testing Performed

Manual testing
Screen Shot 2022-07-06 at 7 19 16 PM

Screen Shot 2022-07-06 at 7 22 57 PM

Screen Shot 2022-07-06 at 7 23 04 PM

### CODE REVIEW

**Review by sachaudh — APPROVED** [review]
LGTM

File: ui/apps/platform/src/Containers/VulnMgmt/Entity/TableWidgetFixableCves.js [hunk 1]
```diff
@@ -30,7 +31,14 @@ const queryFieldNames = {
...
 };
 
-const TableWidgetFixableCves = ({ workflowState, entityContext, entityType, name, id }) => {
+const TableWidgetFixableCves = ({
+    workflowState,
+    entityContext,
+    entityType,
+    name,
+    id,
+    vulnType = entityTypes.CVE,
```
Comment (sachaudh) [inline ui/apps/platform/src/Containers/VulnMgmt/Entity/TableWidgetFixableCves.js:40]: In the Deployments, Namespace, and Node sections, I think we would need to pass the correct vulnType, no?

Comment (vjwilson) [reply ui/apps/platform/src/Containers/VulnMgmt/Entity/TableWidgetFixableCves.js:40]: Correct. I deferred that--can be done in another PR.

**Comment by vjwilson** [comment]
This looks like it really broke a few ui-e2e tests.

**Comment by sachaudh** [comment]
Yea, I'm not sure how useful it is to work with the flag on for CI. It'll just feel like a game of Whack-a-Mole. Would be reasonable to turn it off in CI and use that as an indicator that things still work with any changes we make for VM, and then near the end try to make changes in our tests to adapt to whether the feature flag is on or off?

**Comment by sachaudh** [comment]
Oh sweet, @vjwilson I think I fixed the failing tests in this PR just by increasing the timeout for the API calls https://github.com/stackrox/stackrox/pull/2314

**Pull request merged**